.github/scripts/test_trymerge.py

```diff
@@ -0,0 +1,28 @@
'author': {'user': None, 'email': 'ghp_XXXXXXXXXXXXX', 'name': 'coolteemf'}, 'oid': '9d81d7bae8ad91aaa24b3ceab83e3138894dbc69', 'checkSuites': {'nodes': [{'app': {'databaseId': 12274}, 'conclusion': 'SUCCESS'}]}}}, {'commit': {'author': {'user': None, 'email': 'ghp_XXXXXXXXXXXXX', 'name': 'coolteemf'}, 'oid': 'e79f6a2202512b294c55bf4bfb2e0524fafd4c48', 'checkSuites': {'nodes': [{'app': {'databaseId': 12274}, 'conclusion': 'SUCCESS'}]}}}, {'commit': {'author': {'user': None, 'email': 'ghp_XXXXXXXXXXXXX', 'name': 'coolteemf'}, 'oid': 'f683e8aec7aea76097a264eec01511e704c31154', 'checkSuites': {'nodes': [{'app': {'databaseId': 12274}, 'conclusion': 'SUCCESS'}]}}}, {'commit': {'author': {'user': {'login': 'coolteemf'}, 'email': '67541941+coolteemf@users.noreply.github.com', 'name': 'François Lecomte'}, 'oid': 'b932e9e286c22aaf352375186df851ef060b295a', 'checkSuites': {'nodes': [{'app': {'databaseId': 12274}, 'conclusion': 'SUCCESS'}]}}}, {'commit': {'author': {'user': None, 'email': 'ghp_XXXXXXXXXXXXX', 'name': 'coolteemf'}, 'oid': '346e0c547953d98eb84d23c1391a95badb9c4a22', 'checkSuites': {'nodes': [{'app': {'databaseId': 12274}, 'conclusion': 'SUCCESS'}]}}}], 'totalCount': 16}, 'changedFiles': 9, 'files': {'nodes': [{'path': 'aten/src/ATen/native/GridSampler.cpp'}, {'path': 'aten/src/ATen/native/cpu/GridSamplerKernel.cpp'}, {'path': 'aten/src/ATen/native/cuda/GridSampler.cpp'}, {'path': 'aten/src/ATen/native/cuda/GridSampler.cu'}, {'path': 'aten/src/ATen/native/cuda/GridSampler.h'}, {'path': 'aten/src/ATen/native/native_functions.yaml'}, {'path': 'test/forward_backward_compatibility/check_forward_backward_compatibility.py'}, {'path': 'test/test_nn.py'}, {'path': 'tools/autograd/derivatives.yaml'}]}, 'reviews': {'nodes': [{'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'coolteemf'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'COMMENTED'}, {'author': {'login': 'albanD'}, 'state': 'APPROVED'}, {'author': {'login': 'albanD'}, 'state': 'APPROVED'}], 'totalCount': 17}, 'comments': {'nodes': [{'bodyText': 'Thanks for the update! The windows failure is not your fault, you can ignore it!\n\nThank you very much for all of your feedback and sorry for the delay !', 'author': {'login': 'coolteemf'}, 'authorAssociation': 'NONE', 'editor': None}]}}}}}
```

Comment:
This is an immense json blob o.O

Reply:
Slightly redacted, as I don't want to leak wrong email here